src/index.js

```diff
@@ -14,6 +14,12 @@ module.exports = function mjml (mjmlEngine, options) {
   }
 
   return through.obj(function (file, enc, callback) {
+    // Not a big fan of this deep copy methods
+    // But it will work regardless of Node version
+    var localOptions = JSON.parse(JSON.stringify(options))
+    if (localOptions.filePath === undefined) {
+      localOptions.filePath = file.path.toString()
```

Comment:
Is this actually doing what it is supposed to? Shouldn't it set `options.filePath = file.path.toString()` instead?

I tried that locally instead, and then includes started working with relative paths.